src/rest/impl/contracts.service.ts

```diff
@@ -139,7 +141,185 @@ export default class ContractsServiceImpl extends ContractsService {
     
     res.status(200).send(await this.translateDatabaseContract(databaseContract));
   }
-  
+
+  /**
+   * @inheritdoc
+   */
+  async importContracts(req: Request, res: Response) {
+    if (!this.hasRealmRole(req, ApplicationRoles.CREATE_CONTRACT)) {
+      this.sendForbidden(res, "You have no permission to import contracts");
+      return;
+    }
+
+    try {
+      const importFile = req.file;
```

Comment:
Should the header row be checked for validity?

Reply:
No need to validate header row since it is not used anywhere, the file structure needs to be correct from the second row. First row is reserved for header texts just for human convenience when filling data to file.